deploy/kubernetes/console/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+Thank you for installing Stratos. Your release is named '{{ .Release.Name }}' and is in the namespace '{{ .Release.Namespace }}'.
```

Comment:
We should look at some other good charts like Prometheus for inspiration as to what to show.